Common/DataModel/Centrality_new.h

```diff
@@ -0,0 +1,35 @@
+// Copyright 2019-2020 CERN and copyright holders of ALICE O2.
```

Comment:
When we merge this, we will replace `Centrality.h` right away (and all uses of it of course)...

Reply:
Correct!
I did not do it because I had to change all usages of the `Cents` table
Perhaps we can do that as a previous step?

Reply:
We can do this in bulk once this is ready

Reply:
OK!